docs/content/docs/contributing/Docs.md

```diff
@@ -307,27 +307,29 @@ Markdown content
 
 ### Lists
 
-#### Нумерованный или ненумерованный
+#### Ordered vs Unordered lists
 
-Используй нумерованный список только если он описывает последовательность шагов.
+Only use ordered list for describing a series of steps to follow.
```

Comment:
```suggestion
Use ordered list only for describing a series of steps to follow.
```

Reply:
This is intentional. I focus on it.

https://dictionary.cambridge.org/grammar/british-grammar/only (Ordering)